packages/api/src/controllers/userFarmController.js

```diff
@@ -210,8 +210,8 @@ const userFarmController = {
           .where({ user_id, farm_id })
           .patch({ has_consent, consent_version });
         res.sendStatus(200);
-        const { step_two_end, step_three_end, step_five_end } = userFarm;
-        const isWelcomeEmailSent = !!step_two_end && !!step_three_end && !step_five_end;
+        const { step_two_end, step_three_end } = userFarm;
+        const isWelcomeEmailSent = !!step_two_end && !!step_three_end;
```

Comment:
shouldnt it be an "or" operator? `!!step_two_end || !!step_three_end;`

Reply:
I think not, this endpoint triggers in the onboarding flow after you select a role (which is step 2) right at the point you accept the policy  step 2 is complete but step 3 isn't. So the email should be sent exactly then. 

When this condition is met again (when you accept consent because we introduced a new policy) you will have both steps finished.